project/huang-me/README.md

```diff
@@ -0,0 +1,37 @@
+# TSP with gene algorithm
+
+## Basic information
+TSP (Travelling salesman problem) is a problem proved to be NP-hard, so we cannot use brute force to find the optimal solution.
```

Comment:
It's a computer algorithm that can be handled by using integer only.  Could you indicate where you plan to take advantage of (real-number) numerical calculations?

Reply:
Since genetic algorithm need some random number to generate the next state, in addition to multi-thread, I'm going to use some other random number generate algorithm to speedup the process.

Reply:
Do you plan to use a true random number generator (RNG) or a pseudo random number generator (PRNG)?  A true RNG will need hardware support.  Writing a PRNG is a lot of work and may constitute a project by itself.

What thing do you want to focus on in this project?  RNG, PRNG, or TSP?

Reply:
I'd like to focus on speeding up genetic algorithm for TSP problem by numerical calculation, like calculating cost in a faster way (hashing the route cost) or implementing pseudorandom number generator.

Is that okay for the final project?